web/src/ButtonInput.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import styled from "styled-components"
+import { Button } from "./Button"
+
+let ButtonInputRoot = styled.input`
+  ${Button}
+  border: 0;
+  width: 100%;
+`
+
+type ButtonLinkProps = {
+  value: string
+  type: string
+}
+
+function ButtonLink(props: ButtonLinkProps) {
```

Comment:
I got really confused that this file is called ButtonInput but the class is called ButtonLink